### PR TITLE
Mobile users can see detailed content

### PR DESCRIPTION
**Pull request by mantanquzzmo**
[Mobile users can see detailed content](https://www.pivotaltracker.com/story/show/170925619)
[Mobile users log in](https://www.pivotaltracker.com/story/show/170925679)

As a visitor
In order to read my specific article of choice
I want to be able to read that article on the screen

As a visitor
In order use the full services of the mobile app
I want to be able to log in

- Adds show page functionality
- Adds login functionality

### CODE REVIEW

File: src/modules/article.js [hunk 1]
```diff
@@ -0,0 +1,114 @@
+import axios from "axios";
+
```
Comment (oliverochman) [inline src/modules/article.js]: there seems t be a lot of methods in this module that are out of scope?

File: src/pages/ShowArticle.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          </IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      {currentArticle && (
+        <IonContent>
+          <IonItem class="show-article">
+            <IonText>
+              <h3>{currentArticle.title}</h3>
+              {currentArticle.body}
+              <img src={currentArticle.image}></img>
```
Comment (oliverochman) [inline src/pages/ShowArticle.jsx:62]: perhaps say that you are not showing the whole article body because the user is not a subscriber if that is the case

**Review by oliverochman — CHANGES_REQUESTED** [review]

File: src/modules/article.js (hunk 1 above)
Comment (mantanquzzmo) [reply src/modules/article.js]: Yes, I seem to accidentally have done two of the features in one.

File: src/pages/ShowArticle.jsx (hunk 2 above)
Comment (mantanquzzmo) [reply src/pages/ShowArticle.jsx:62]: Fixed

**Review by oliverochman — APPROVED** [review]
